apps/wide-deep-recommendation/dummy_data_generating.py

```diff
@@ -0,0 +1,144 @@
+                        help='The path for output dataset')
+    args = parser.parse_args()
+    return args
+
+
+def generate_record(random_seed):
+    random.seed(random_seed)
+    test_tokens = "\t".join([str(random.randint(1, 1000))
+                            for i in range(random.randint(1, 10))])
+    hashtags = "\t".join(["".join(random.choices(id_list, k=32))
+                          for i in range(random.randint(0, 50))])
+    tweet_id = "".join(random.choices(id_list, k=32))
+    present_media = "\t".join(random.choices(
+        media_list, k=random.randint(0, 9)))
+    present_links = "\t".join(["".join(random.choices(id_list, k=32))
+                               for i in range(random.randint(0, 10))])
+    present_domains = "\t".join(["".join(random.choices(id_list, k=32))
+                                for i in range(random.randint(0, 10))])
+    tweet_type = random.choices(tweet_list)[0]
+    language = random.choices(language_list)[0]
+    tweet_timestamp = random.randint(946656000, 1609430400)
+    engaged_with_user_id = "".join(random.choices(id_list, k=32))
+    engaged_with_user_follower_count = random.randint(0, 10000)
+    engaged_with_user_following_count = random.randint(0, 10000)
+    engaged_with_user_is_verified = bool(random.getrandbits(1))
+    engaged_with_user_account_creation = random.randint(946656000, 1609430400)
+    enaging_user_id = "".join(random.choices(id_list, k=32))
+    enaging_user_follower_count = random.randint(0, 10000)
+    enaging_user_following_count = random.randint(0, 10000)
+    enaging_user_is_verified = bool(random.getrandbits(1))
+    enaging_user_account_creation = random.randint(946656000, 1609430400)
+    engagee_follows_engager = bool(random.getrandbits(1))
+    reply = bool(random.getrandbits(1))
+    reply_timestamp = random.randint(946656000, 1609430400) if reply else None
+    retweet = bool(random.getrandbits(1))
+    retweet_timestamp = random.randint(
+        946656000, 1609430400) if retweet else None
+    comment = bool(random.getrandbits(1))
+    retweet_with_comment_timestamp = random.randint(
+        946656000, 1609430400) if comment else None
+    like = bool(random.getrandbits(1))
+    like_timestamp = random.randint(946656000, 1609430400) if like else None
+    return (test_tokens, hashtags, tweet_id, present_media, present_links, present_domains,
+            tweet_type, language, tweet_timestamp, engaged_with_user_id,
+            engaged_with_user_follower_count, engaged_with_user_following_count,
+            engaged_with_user_is_verified, engaged_with_user_account_creation,
+            enaging_user_id, enaging_user_follower_count, enaging_user_following_count,
+            enaging_user_is_verified, enaging_user_account_creation,
+            engagee_follows_engager, reply_timestamp, retweet_timestamp,
+            retweet_with_comment_timestamp, like_timestamp)
+
+
+if __name__ == '__main__':
+    args = _parse_args()
+
+    OrcaContext.log_output = True
```

Comment:
remove this, this is only needed for notebook
Also import OrcaContext can be removed as well?

Reply:
This script uses a spark session to create a data frame, and we get the session from OrcaContext.get_spark_session. I get a way to get the session from the spark context directly, how about this?
```
val sc = new SparkContext(conf)
val spark = SparkSession.builder.config(sc.getConf).getOrCreate()
```